src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -290,6 +295,6 @@ protected function verifyMigrationDirectory($path)
      */
     protected function getMigrationTemplate()
     {
-        return file_get_contents(dirname(__FILE__) . '/../../Migration/Migration.template.php.dist');
+        return file_get_contents(dirname(__FILE__) . self::DEFAULT_MIGRATION_TEMPLATE);
```

Comment:
Always use `static` instead of `self`, to allow extending classes to easily overload.